From 9589a12abae987656132da8cc59ac7b3a6fdf65e Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Tue, 12 Aug 2014 11:30:56 -0400
Subject: [char] hwrng: virtio - remove unused struct element

Message-id: <5117fefa1c899b5bff8f13e42da0cf8f7b579b6c.1407842211.git.amit.shah@redhat.com>
Patchwork-id: 87653
O-Subject: [RHEL7.1 kernel PATCH 6/9] virtio: rng: remove unused struct element
Bugzilla: 1119299
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Herbert Xu <herbert.xu@redhat.com>

vdev is unused in struct virtrng_info, remove it.

CC: Amos Kong <akong@redhat.com>
Signed-off-by: Amit Shah <amit.shah@redhat.com>
Signed-off-by: Rusty Russell <rusty@rustcorp.com.au>
(cherry picked from commit 373445d02befffe2efe31ebf1ab7e566d877e3ee)
Signed-off-by: Amit Shah <amit.shah@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/char/hw_random/virtio-rng.c b/drivers/char/hw_random/virtio-rng.c
index 4b10345..5799389 100644
--- a/drivers/char/hw_random/virtio-rng.c
+++ b/drivers/char/hw_random/virtio-rng.c
@@ -29,7 +29,6 @@
 static DEFINE_IDA(rng_index_ida);
 
 struct virtrng_info {
- struct virtio_device *vdev;
  struct hwrng hwrng;
  struct virtqueue *vq;
  unsigned int data_avail;
-- 
1.7.1